Mathlib/Data/Bool/Basic.lean

```diff
@@ -21,6 +21,123 @@ bool, boolean, Bool, De Morgan
 
 namespace Bool
 
+#align bor or
+#align band and
+#align bnot not
+#align bxor xor
+
+section
+
+/-!
+This section contains lemmas about booleans which were present in core Lean 3.
+The remainder of this file contains lemmas about booleans from mathlib 3.
```

Comment:
I think "of this file" is no longer correct

Reply:
I think it is! I adapted the comment when moving it to this file, because initially it instructed readers to look in `Mathlib/Data/Bool/Basic.lean` for lemmas coming from mathlib 3.